nvflare/private/fed/server/server_engine_internal_spec.py

```diff
@@ -237,41 +245,33 @@ def ask_to_stop(self):
         pass
 
     @abstractmethod
-    def aux_send(self, targets: [], topic: str, request: Shareable, timeout: float, fl_ctx: FLContext) -> dict:
-        """Send a request to client(s) via the auxiliary channel.
+    def show_stats(self, job_id) -> dict:
+        """Show_stats of the server.
 
         Args:
-            targets: list of Client or client names
-            topic: topic of the request
-            request: request to be sent
-            timeout: number of secs to wait for replies
-            fl_ctx: FL context
+            job_id: current job_id
 
         Returns:
-             A dict of replies: client_name => Shareable
-
-        NOTE: when a reply is received, the peer_ctx props must be set into the PEER_PROPS header
-        of the reply Shareable.
+            Component stats of the server
 
-        If a reply is not received from a client, do not put it into the reply dict.
         """
         pass
 
     @abstractmethod
-    def show_stats(self, job_id) -> dict:
-        """Show_stats of the server.
+    def get_errors(self, job_id) -> dict:
+        """Get the errors of the server components.
 
         Args:
             job_id: current job_id
 
         Returns:
-            Component stats of the server
+            Server components errors.
 
         """
         pass
 
     @abstractmethod
-    def get_errors(self, job_id) -> dict:
+    def reset_errors(self, job_id) -> str:
```

Comment:
this method is not needed in the spec, please remove it.

Reply:
This will b needed in next PR when adding the reset_error command.